### PR TITLE
Updates tips.

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
Clarifies some tips. Adds some brand new tips. Removes an old tip that is not true anymore.

## Why It's Good For The Game
Tips good.

## Testing
It compiled.

## Changelog
:cl:
add: Adds some new tips.
del: Removes an incorrect tip.
tweak: Tweaks some tips.
/:cl:

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -1,13 +1,13 @@
 Space map levels' connections are randomized between rounds, but are otherwise kept consistent in the same round.
 You can catch thrown items by toggling on your throw mode with an empty hand active.
-To crack the safe in the vault, use a stethoscope or thermal drill.
+To crack the safe in the vault, use a stethoscope or thermal drill. You can also steal the combonation paper from the captain's office, if they didn't secure it.
```
Comment (Scribble-Sheep) [inline strings/tips.txt:3]:
```suggestion
To crack the safe in the vault, use a stethoscope or thermal drill. You can also steal the combination paper from the captain's office, if they didn't secure it.
```

File: strings/sillytips.txt [hunk 2]
```diff
@@ -29,4 +29,5 @@ Absolutely nothing unusual will happen this shift. Promise.
 Send help. Trapped in tip writing factory.
 There is no such thing as a cow level.
 HONK.
-Solars probably gives radiation poisoning or something, considering the other power sources are deadly in their own ways.
\ No newline at end of file
+Solars probably gives radiation poisoning or something, considering the other power sources are deadly in their own ways.
```
Comment (Scribble-Sheep) [inline strings/sillytips.txt:32]:
```suggestion
Solars probably give radiation poisoning or something, considering the other power sources are deadly in their own ways.
```

**Review by Scribble-Sheep — CHANGES_REQUESTED** [review]
Great argument. Unfortunately, I have spotted a minor typo!

File: strings/sillytips.txt (hunk 2 above)
Comment (Scribble-Sheep) [reply strings/sillytips.txt:32]: This one prolly isn't a typo, but it feels like it flows more naturally to me.

**Review by Burzah — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]

**Pull request merged**